middlewares/metrics_test.go

```diff
@@ -3,18 +3,22 @@ package middlewares
 import (
 	"testing"
 
+	"net/http"
+	"net/http/httptest"
+
 	"github.com/go-kit/kit/metrics"
 )
 
 func TestMetricsRetryListener(t *testing.T) {
+	req := httptest.NewRequest(http.MethodGet, "/", nil)
 	// nil implementation, nothing should fail
 	retryListener := NewMetricsRetryListener(nil)
-	retryListener.Retried(1)
+	retryListener.Retried(req, 1)
 
 	retryMetrics := newCollectingMetrics()
 	retryListener = NewMetricsRetryListener(retryMetrics)
-	retryListener.Retried(1)
-	retryListener.Retried(2)
+	retryListener.Retried(req, 1)
+	retryListener.Retried(req, 2)
```

Comment:
In general yes, but in this case we did not expand the SUT. It only gets the request now as parameter, but it doesn't do anything with it. The request was introduced for the access log implementation of the `RetryListener` interface.